go.mod

```diff
@@ -18,4 +18,5 @@ require (
 	golang.org/x/sys v0.14.0 // indirect
 	golang.org/x/text v0.14.0 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20231120223509-83a465c0220f // indirect
+	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0 // indirect
```

Comment:
We now have a `tools.go` file that we add pseudo-imports to to keep the corresponding `go.mod` entries even after running `go mod tidy`. Could you add an import there so that this becomes a direct dep?

Reply:
Good idea. Done.